sdk/template/azure-template/inc/azure/template/version.hpp

```diff
@@ -47,3 +47,8 @@ namespace Azure { namespace Template { namespace Details {
 #undef AZURE_TEMPLATE_VERSION_MINOR
 #undef AZURE_TEMPLATE_VERSION_PATCH
 #undef AZURE_TEMPLATE_VERSION_PRERELEASE
+
+
+
+
+
```

Comment:
remove it maybe?   Are we adding one space automatically?

Reply:
the `Update-PkgVersion.ps1` script is doing it